### PR TITLE
HPCC-13613 Document STORED(FORMAT(PASSWORD))

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@afishbeck Please review

### CODE REVIEW

File: docs/ECLLanguageReference/ECLR_mods/WkFlo-STORED.xml [hunk 1]
```diff
@@ -106,14 +108,22 @@
...
-            displayed on the Web form in WsECL. </entry>
+            displayed on the Web form in WsECL.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">PASSWORD</emphasis></entry>
+
+            <entry>Optional. If PASSWORD is specified, a password entry box is
+            used on the Web form in WsECL and the field's supplied value is
+            not displayed while entering it.</entry>
```
Comment (afishbeck) [inline docs/ECLLanguageReference/ECLR_mods/WkFlo-STORED.xml]: The value is also hidden when viewing the stored values in the workunit through EclWatch (or command line).  Also when extracting the xml, etc.

**Comment by richardkchapman** [comment]
@JamesDeFabia Back to you

**Comment by JamesDeFabia** [comment]
@afishbeck Edits made as requested, please have another look

**Comment by afishbeck** [comment]
@JamesDeFabia looks good

**Pull request merged**